requirements_no_sql.txt

```diff
@@ -31,3 +31,6 @@ snowballstemmer==1.2.0
 traitlets==4.0.0
 websocket-client==0.34.0
 wsgiref==0.1.2
+corr==0.7.3
+-e git+git://github.com/sma-wideband/adc_tests.git@65a2ef4e1cf68bee35176a1171d923a73952e13e#egg=adc5
```

Comment:
this is solved: the adc_tests package comes from project8/adc_tests (forked from sma-wideband), and has a version control.
A tag (v0.0.1) has been created for version that currently works 65a2ef4e1cf68bee35176a1171d923a73952e13e: the associated version defined in the setup.py is 0.0.1.
In order to prevent confusion, I bumped the latest version in the master branch to 0.0.2 in the setup.py and tagged it. Installing this version with the requirements adc5g==0.0.1 in dragonfly setup.py file, will cause a crash of the installation (recommended...)